public/css/owner.css

```diff
@@ -0,0 +1,50 @@
+.logo-owner{
+    width: 100%;
+    height: 20%;
+    background-color: white;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-around;
+    text-align: center;
+    margin: 0 0 2em 0;
+    border-radius: 0 0 1em 1em;
+}
+
+#button-owner{
+    background-color: lightgray;
+    height: 4em;
+    width: 10em;
```

Comment:
duzo tych em`ow. moze lepiej takie wielkosci podawac w procentach?

Reply:
% coś sie ...., no, w każdym razie testowaliśmy i często jak dawaliśmy % to nic się nie zmieniało, dlatego na em robione w większości